projects/igniteui-angular/src/lib/badge/badge.component.html

```diff
@@ -1,4 +1,2 @@
-<div class="igx-badge__circle" [ngClass]="setClasses()" [attr.aria-roledescription]="roleDescription">
-    <span *ngIf="!icon" class="igx-badge__circle-value">{{value}}</span>
-    <igx-icon *ngIf="icon" fontSet="material">{{icon}}</igx-icon>
-</div>
+<span *ngIf="!icon" class="igx-badge__circle-value">{{value}}</span>
```

Comment:
```suggestion
<span *ngIf="!icon" class="igx-badge__value">{{value}}</span>
```
This no longer needs the`__circle` part in the name.